Remotion/Validation/UnitTests/Implementation/InvariantValidationMessageTest.cs

```diff
@@ -14,29 +14,53 @@
 // You should have received a copy of the GNU Lesser General Public License
 // along with re-motion; if not, see http://www.gnu.org/licenses.
 // 
+using System;
+using System.Globalization;
 using NUnit.Framework;
+using Remotion.Validation.Implementation;
 
 namespace Remotion.Validation.UnitTests.Implementation
 {
   [TestFixture]
   public class InvariantValidationMessageTest
   {
     [Test]
-    [Ignore("RM-5906")]
-    public void Format_UsesSuppliedFormatProviderInsteadOfCurrentCulture ()
+    [SetCulture("en-US")]
+    [SetUICulture("de-CH")]
+    public void Format_WithFormatProvider_UsesSuppliedFormatProviderInsteadOfCurrentCulture ()
     {
+      var validationMessage = new InvariantValidationMessage("{0} : {1}");
+      var culture = CultureInfo.GetCultureInfo("fr-FR");
+      var formatProvider = CultureInfo.GetCultureInfo("de-AT");
+      var parameters = new object[] { 3123.312, new DateTime(2023, 05, 15) };
+
+      var result = validationMessage.Format(culture, formatProvider, parameters);
+
+      Assert.That(result, Is.EqualTo("3123,312 : 15.05.2023 00:00:00"));
     }
 
     [Test]
-    [Ignore("RM-5906")]
-    public void Format_UsesSuppliedCultureInsteadOfCurrentUICulture ()
+    [SetCulture("en-US")]
+    [SetUICulture("de-CH")]
+    public void Format_WithNullAsFormatProvider_UsesSuppliedCultureInsteadOfCurrentUICulture ()
     {
+      var validationMessage = new InvariantValidationMessage("{0} : {1}");
+      var culture = CultureInfo.GetCultureInfo("fr-FR");
+      var parameters = new object[] { 3123.312, new DateTime(2023, 05, 15) };
+
+      var result = validationMessage.Format(culture, null, parameters);
+
+      Assert.That(result, Is.EqualTo("3123.312 : 05/15/2023 00:00:00"));
     }
 
     [Test]
-    [Ignore("RM-5906")]
-    public void ToString_Override ()
+    public void ToString_ReturnsValidationMessage ()
     {
+      var validationMessage  = new InvariantValidationMessage("test");
+
+      var result = validationMessage.ToString();
+
+      Assert.That(result, Is.EqualTo("test"));
```

Comment:
```suggestion
      var validationMessage  = new InvariantValidationMessage("test {0} message");

      var result = validationMessage.ToString();

      Assert.That(result, Is.EqualTo("test {0} message"));
```

By adding a placeholder in the format string, we show that ToString will not do anything with it. (otherwise, there would be an exception)